src/main/plugin/iso19139.rndt/schema/gmd/citation.xsd

```diff
@@ -51,7 +51,7 @@
 					<xs:element name="edition" type="gco:CharacterString_PropertyType" minOccurs="0"/>
 					<xs:element name="editionDate" type="gco:Date_PropertyType" minOccurs="0"/>
 					<xs:element name="identifier" type="gmd:MD_Identifier_PropertyType" minOccurs="0" maxOccurs="unbounded"/>
-					<xs:element name="citedResponsibleParty" type="gmd:CI_ResponsibleParty_PropertyType" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="citedResponsibleParty" type="gmd:CI_ResponsibleParty_PropertyType" minOccurs="1" maxOccurs="unbounded"/>
```

Comment:
This is not a Good Thing!

We don't want resp parties for a thesaurus, for instance

Reply:
fixed